edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -44,6 +45,12 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            // Seperate metrics endpoint from https authentication
+            app.Map("/metrics", metricsApp =>
+            {
+                metricsApp.UseMetricServer(string.Empty);
```

Comment:
Is this a prometheus-net extension? Can we hide behind one of our types?

Reply:
same as above